### ISSUE TITLE
Invalid trailing comma with multiline typealias

### ISSUE THREAD

**Issue by keith** [nicklockwood/SwiftFormat#1179]
```
% swiftformat --version
0.49.8
```

```swift
func foo() {
    typealias SomeAlias = [
        Key: Value
    ]
}
```

`swiftformat foo.swift`:

```swift
func foo() {
    typealias SomeAlias = [
        Key: Value,
    ]
}
```

**Comment by nicklockwood** [comment]
@keith fixed in 0.49.9

**Comment by keith** [comment]
Thanks!
